akka-http-core/src/main/scala/akka/http/impl/util/LogByteStringTools.scala

```diff
@@ -17,14 +17,14 @@ private[akka] object LogByteStringTools {
 
   def logByteStringBidi(name: String, maxBytes: Int = MaxBytesPrinted): BidiFlow[ByteString, ByteString, ByteString, ByteString, NotUsed] =
     BidiFlow.fromFlows(
-      logByteString(s"$name DOWN", maxBytes),
-      logByteString(s"$name UP  ", maxBytes)
+      logByteString(s"$name FromNet", maxBytes),
+      logByteString(s"$name ToNet  ", maxBytes)
     )
 
   def logToStringBidi[A, B](name: String, maxBytes: Int = MaxBytesPrinted): BidiFlow[A, A, B, B, NotUsed] =
     BidiFlow.fromFlows(
-      logToString(s"$name DOWN", maxBytes),
```

Comment:
Same as for `logByteStringBidi`, generic names were better here as we don't know how it is used.

Reply:
Or, you could add a `ClassTag` for `A` and `B` and use some string from those here.

Reply:
good idea, done, with limiting it to some chars